hack/common.sh

```diff
@@ -94,7 +94,15 @@ deploy_image_to_cluster() {
 	    minikube cache add $image
 	    ;;
 	'kind')
-	   kind load docker-image $image --name $clustername
+        if [[ "${LOCAL_CONTAINER_ENGINE}" == "podman" ]];
+        then
+            echo_green "Workaround for kind on podman, see https://github.com/kubernetes-sigs/kind/issues/2027"
+            podman save "$image" > temp.tar
```

Comment:
@andreadecorte  since we're here mind stretch the work to do this?

```
tmpfile=$(mktemp --suffix .tar)
podman save "$image" > $tmpfile
```

and then
```
rm $tmpfile
```